app/models/satisfaction_survey_feature_metrics.rb

```diff
@@ -42,6 +60,16 @@ def application_forms_with_feedback(start_time, end_time)
       .where(recruitment_cycle_year: RecruitmentCycle.current_year)
   end
 
+  def application_forms_with_positive_feedback(start_time, end_time)
+    ApplicationForm
+      .where(
+        recruitment_cycle_year: RecruitmentCycle.current_year,
+      )
+      .where('application_forms.submitted_at BETWEEN ? AND ?', start_time, end_time)
+      .where(feedback_satisfaction_level: 'very_satisfied').or(ApplicationForm.where(feedback_satisfaction_level: 'satisfied'))
```

Comment:
I think this should generate a `WHERE x IN (a,b)` query, which does the same thing as an OR:

```suggestion
      .where(feedback_satisfaction_level: ['very_satisfied', 'satisfied'])
```

Reply:
I'll take a 👀

Reply:
:+1: it will, and it will get around the issue that I highlighted with brackets and `OR`.